src/app/core/models/label.model.ts

```diff
@@ -2,23 +2,15 @@
  * Represents a label and its attributes.
  */
 export class Label implements SimpleLabel {
-  readonly category: string;
-  readonly name: string;
-  readonly formattedName: string; // 'category'.'name' (e.g. severity.Low) if a category exists or 'name' if the category does not exist.
+  readonly formattedName: string;
```

Comment:
Wondering if we should refactor all instances of `label.formattedName` to just `label.name` to prevent confusion since we now aren't doing any formatting of the name.

Reply:
Agree

Reply:
I have changed `label.formattedName` to `label.name`. Do let me know if there are any other issues!